core/src/css/palettes/high-contrast-dark.scss

```diff
@@ -128,10 +128,6 @@ $lightest-text-color: $text-color;
     --ion-text-color-rgb: #{color-to-rgb-list($text-color)};
     --ion-item-background: #000000;
     --ion-card-background: #1c1c1d;
-
-    /// Only the item borders should increase in contrast
-    /// Borders for elements like toolbars should remain the same
-    --ion-item-border-color: var(--ion-background-color-step-400);
```

Comment:
These have been moved to the modal file along with making a components variable under the high contrast dark theme.